apps/client/src/domains/routing/Routing.tsx

```
@@ -0,0 +1,44 @@
+import { Engine } from '@common/schemas/routing';
+
+import { useStore } from '@/shared/services';
+import { Error } from '@/shared/ui';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)